google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/appengine/cloud/flexible/AppEngineFlexibleStageTest.java

```diff
@@ -96,6 +96,18 @@ public void testStage_noDockerDirectory() {
     }
   }
 
+  public void testStage_noModule() {
+    deploymentConfiguration.setModuleName(null);
+    AppEngineFlexibleStage stage =
+        new AppEngineFlexibleStage(loggingHandler, artifact, deploymentConfiguration, getProject());
+    try {
+      stage.stage(stagingDirectory);
+      fail("expected failure due to missing module name in deployment config");
+    } catch (RuntimeException re) {
+      assertEquals("Error occurred during App Engine flexible staging: no app.yaml configuration file was specified.", re.getMessage());
```

Comment:
Thought so too.. But since the formatter doesn't do this, maybe its not in the official style. I'm not bothered by it so I'll just go ahead and update them